webapp/src/types/store.ts

```diff
@@ -1,26 +1,11 @@
 import {GlobalState as ReduxGlobalState} from 'mattermost-redux/types/store';
 
-import {Item} from './gitlab_items';
-
-import {GitlabUsersData, LHSData, ShowRhsPluginActionData, UserSettingsData} from '.';
+import combinedReducers from '../reducers';
```

Comment:
If we want avoid other files transitively importing the code of the `reducers` file, we can change this to a type import

```suggestion
import type combinedReducers from '../reducers';
```